cmd/init.go

```diff
@@ -109,90 +111,92 @@ var initCmd = &cli.Command{
 	Usage: "create a new gqlgen project",
 	Flags: []cli.Flag{
 		&cli.BoolFlag{Name: "verbose, v", Usage: "show logs"},
-		&cli.StringFlag{Name: "config, c", Usage: "the config filename"},
+		&cli.StringFlag{Name: "config, c", Usage: "the config filename", Value: "gqlgen.yml"},
 		&cli.StringFlag{Name: "server", Usage: "where to write the server stub to", Value: "server.go"},
 		&cli.StringFlag{Name: "schema", Usage: "where to write the schema stub to", Value: "graph/schema.graphqls"},
 	},
 	Action: func(ctx *cli.Context) error {
 		configFilename := ctx.String("config")
 		serverFilename := ctx.String("server")
+		schemaFilename := ctx.String("schema")
 
 		pkgName := code.ImportPathForDir(".")
 		if pkgName == "" {
 			return fmt.Errorf("unable to determine import path for current directory, you probably need to run go mod init first")
 		}
 
-		if err := initSchema(ctx.String("schema")); err != nil {
-			return err
+		// check schema and config don't already exist
+		if fileExists(configFilename) {
+			return fmt.Errorf("%s already exists\n", configFilename)
+		}
+		if fileExists(schemaFilename) {
+			return fmt.Errorf("%s already exists\n", schemaFilename)
 		}
-		if !configExists(configFilename) {
-			if err := initConfig(configFilename, pkgName); err != nil {
-				return err
-			}
+		if fileExists(serverFilename) {
+			return fmt.Errorf("%s already exists\n", serverFilename)
+		}
+		_, err := config.LoadConfigFromDefaultLocations()
+		if err == nil {
+			return fmt.Errorf("gqlgen.yml already exists in a parent directory\n")
 		}
 
-		GenerateGraphServer(serverFilename)
-		return nil
-	},
-}
+		// create config
+		fmt.Println("Creating", configFilename)
+		if err := initFile(configFilename, executeConfigTemplate(pkgName)); err != nil {
+			return err
+		}
 
-func GenerateGraphServer(serverFilename string) {
-	cfg, err := config.LoadConfigFromDefaultLocations()
-	if err != nil {
-		fmt.Fprintln(os.Stderr, err.Error())
-	}
+		// create schema
+		fmt.Println("Creating", schemaFilename)
```

Comment:
Can we enable this printing only on debug/verbose mode?  Some users (like myself), find it more annoying than useful.

Reply:
Yep I'm a big fan of the "[silence is golden](http://www.linfo.org/rule_of_silence.html)" philosophy, and follow it myself all the time for CLI tools - especially in tools like `aws-vault` which I maintain.

Yet in this case I think we should actually be more verbose when running `gqlgen init` for the following reasons
 1. `gqlgen init` is the first command a user will run when using gqlgen. Many users won't be familiar with it or what it's doing. 
 2. `gqlgen init` only gets run once. It's not a command users will regularly use.
 3. `gqlgen init` is a slightly "surprising" command - it only creates boilerplate config and schemas, that should then be immediately edited to fit the use-case. Creating that visibility makes that fact more obvious
 4. It mirrors the behaviour of `go mod init` which lists the files that it creates. `go mod init` is the command that will precede `gqlgen init` most of the time

I think the "silence is golden" philosophy may be better applied to `gqlgen generate` where none of the above is true